frontend/src/app/main/data/workspace/versions.cljs

```diff
@@ -148,6 +148,29 @@
                                  (fetch-versions)
                                  (ptk/event ::ev/event {::ev/name "pin-version"})))))))))
 
+(defn lock-version
+  [id]
+  (assert (uuid? id) "expected valid uuid for `id`")
+  (ptk/reify ::lock-version
+    ptk/WatchEvent
+    (watch [_ _ _]
+      (->> (rp/cmd! :lock-file-snapshot {:id id})
+           (rx/map fetch-versions)
+           (rx/catch (fn [error]
+                       (js/console.error "Failed to lock version:" error)
```

Comment:
This will hide the error and only show it on devtools. I suggest remove the rx/catch and let the error escalate to the error handling which will show a proper error message.